### PR TITLE
22 simulink to herbert

### PR DESCRIPTION
**Pull request by abuts**
The task appeared to be much simple as `which` command in Matlab apparently returns realpath. 
This is small changes I had to do doing release. 
Similar changes are in Horace and this request is pared with the Horace one.

### CODE REVIEW

File: herbert_core/configuration/@herbert_config/private/process_unit_test_path.m [hunk 1]
```diff
@@ -9,6 +9,13 @@
         'herbert_init is not on Matlab search path');
 end
 tests_path = fullfile(rootpath,'_test');
+if ~(exist(test_path,'dir')==7)
```
Comment (nickbattam-tessella) [inline herbert_core/configuration/@herbert_config/private/process_unit_test_path.m]: what is this magic `7`?

Comment (abuts) [reply herbert_core/configuration/@herbert_config/private/process_unit_test_path.m]: That's Matlab code for directory. file would be 2

Comment (abuts) [reply herbert_core/configuration/@herbert_config/private/process_unit_test_path.m]: Matlab is relatively old languge, so has number of such gems lying around.

Comment (nickbattam-tessella) [reply herbert_core/configuration/@herbert_config/private/process_unit_test_path.m]: it would be useful to put that in as comment for future record

**Pull request merged**